.github/renovate.json

```diff
@@ -1,63 +1,142 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base",
+    "config:recommended",
     ":disableDependencyDashboard",
     ":semanticCommitsDisabled"
   ],
-  "enabledManagers": ["npm", "regex"],
+  "prHourlyLimit": 0,
   "postUpdateOptions": ["yarnDedupeHighest"],
   "packageRules": [
     {
       "matchDatasources": ["npm"],
       "addLabels": ["javascript"],
-      "stabilityDays": 3,
+      "minimumReleaseAge": "3 days",
       "reviewers": ["team:sig-ux"]
     },
     {
       "matchPackageNames": ["node"],
       "allowedVersions": "/20.[0-9]+.[0-9]+(.[0-9]+)?$/"
+    },
+    {
+      "description": "Should be upgraded in lockstep in order to keep their corresponding Jetty versions aligned, could be grouped but releases are likely separated by a bit of time",
+      "matchManagers": ["maven"],
+      "enabled": false,
+      "matchPackageNames": [
+        "/(org.eclipse.jetty:jetty-maven-plugin|org.jenkins-ci:winstone)/"
+      ]
+    },
+    {
+      "description": "Provided by Jetty and should be aligned with the version provided by the version of Jetty we deliver. See: https://github.com/jenkinsci/jenkins/pull/5211",
+      "matchManagers": ["maven"],
+      "enabled": false,
+      "matchPackageNames": ["/jakarta.servlet:jakarta.servlet-api/"]
```

Comment:
```suggestion
      "matchPackageNames": ["jakarta.servlet:jakarta.servlet-api"]
```
why regex?

Reply:
the package validator did that when adapting from old syntax to new